src/main/java/com/hackclub/hccore/advancements/MileAdv.java

```diff
@@ -0,0 +1,34 @@
+package com.hackclub.hccore.advancements;
+
+import com.fren_gor.ultimateAdvancementAPI.advancement.Advancement;
+import com.fren_gor.ultimateAdvancementAPI.advancement.BaseAdvancement;
+import com.fren_gor.ultimateAdvancementAPI.advancement.display.AdvancementDisplay;
+import com.fren_gor.ultimateAdvancementAPI.advancement.display.AdvancementDisplayBuilder;
+import com.fren_gor.ultimateAdvancementAPI.util.AdvancementKey;
+import com.fren_gor.ultimateAdvancementAPI.util.CoordAdapter;
+import com.hackclub.hccore.HCCorePlugin;
+import org.bukkit.Material;
+import org.bukkit.Statistic;
+import org.bukkit.event.player.PlayerStatisticIncrementEvent;
+
+public class MileAdv extends BaseAdvancement {
+    static AdvancementDisplayBuilder<AdvancementDisplay.Builder, AdvancementDisplay> displayBuilder =
+            new AdvancementDisplay.Builder(Material.ELYTRA, "I'm Gonna Be")
```

Comment:
if this is changed to walking, does this need a new icon?
maybe iron / diamond boots